lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -42,14 +42,14 @@ import { PerformanceEvents } from "../telemetry/performance/PerformanceEvent";
  * Class that handles response parsing.
  */
 export class ResponseHandler {
-    private clientId: string;
-    private cacheStorage: CacheManager;
-    private cryptoObj: ICrypto;
-    private logger: Logger;
-    private homeAccountIdentifier: string;
-    private serializableCache: ISerializableTokenCache | null;
-    private persistencePlugin: ICachePlugin | null;
-    private performanceClient?: IPerformanceClient;
+    private readonly clientId: string;
+    protected readonly cacheStorage: CacheManager;
```

Comment:
I think we can avoid this with the cache changes made by Thomas.

Reply:
Yeah none of this should be needed. I understand this may be a stopgap but all of this will be superseded by the 1P changes and we'll then have to revert all of this again. Would prefer to just hold out until the 1P changes are done